lambda/config.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
@manojlds  My intent / thought was for someone to be able to clone the repo, and apply and deploy this via terraform without having to change or override any of the default values.

I think if someone is at the point of utilizing it as a terraform module, they would be more than comfortable overriding the default values.  That's my thought on it, but honestly it just boils down to preference.